docs/shared/reference/releases.md

```diff
@@ -0,0 +1,39 @@
+# Releases
+
+Major Nx versions are released as the _latest_ every six months, typically around April and October.
+After each major version release, the _previous_ major version moves to long-term support (LTS) for 12 months, after
+which it becomes unsupported.
+Each major version has 18 total months of support from when it is first released to when it falls out of LTS.
+Workspaces should only use major versions that are under LTS.
```

Comment:
Mm... this sentence is weird to me. Obviously, people should be on the latest version? :sweat_smile:  It's also weird because the current version is not LTS yet?

In the same vein, we can encourage people to keep up to date with Nx by saying something along the lines of "Use `nx migrate` often to stay up to date with Nx.